models/user.js

```diff
@@ -1,7 +1,39 @@
+//Schema for User
+const mongoose = require('mongoose');
 
+//Connect to data base
+mongoose.connect('mongodb://localhost/users', { useNewUrlParser: true })
+  .then(() => console.log(`Conneted to the Database...`))
+  .catch((err) => console.log(err));
 
+const userSchema = new mongoose.Schema({
+	name: {
+		type: String,
+		required: true,
+	},
+	password: {
+		type: String,
+		required: true,
+		minlength: 6,
+	},
+	email: {
+		type: String,
+		required: true,
+	},
+	private_chat: {
+		type: Object,
+		default: {},
+	},
+	group_chat: {
+		type: Object,
```

Comment:
check this link https://mongoosejs.com/docs/guide.html for the proper definition of a schema when the type is an Object